SingularityUI/app/actions/log.es6

```diff
@@ -174,7 +174,7 @@ export const taskGroupFetchNext = taskGroupId =>
   function(dispatch, getState) {
     let {tasks, taskGroups, logRequestLength, maxLines} = getState();
 
-    let taskGroup = taskGroups[taskGroupId];
+    const taskGroup = taskGroups[taskGroupId];
```

Comment:
Yeah, I tried to go in and fix the linting errors around the code that I was modifying. I didn't get all of them though. For example, the one on line 175 there is strange. `tasks` is modified later, but the other three items in the let are not. So it doesn't like that it's `let`, because three items aren't changed later. But it can't be `const` because one is. And using a mixture would involve repeating the `getState()` call for no other reason.